modules/pel/peltool/peltool.py

```diff
@@ -179,6 +179,18 @@ def buildOutput(sections: list, out: OrderedDict):
             counts[name][1] = modifier + 1
 
 
+def prettyPrint(Mdata: str) -> str:
+    lines = Mdata.split("\n")
+    for i in range(len(lines)):
+        line = lines[i]
+        if( "\":" in line and "{" not in line ):
```

Comment:
For better readability, we could add some sample input and output in the comment lines about this algorithm.

Reply:
Resolved!